src/components/search/GlobalSearchField.js

```diff
@@ -649,9 +649,19 @@ function GlobalSearchField(props) {
             onKeyDown={(e) => {
                 if (e.key === "Enter") {
                     setOpen(false);
-                    router.push(
-                        `/${NavigationConstants.SEARCH}?searchTerm=${searchTerm}&filter=${filter}`
-                    );
+                    if (filter !== searchConstants.ALL) {
```

Comment:
There's another check below to see if the filter is set to `ALL`.  I think we still want users to be directed to the search page when they hit the `Enter` key, so perhaps this check is unnecessary?

Reply:
Good Catch. Indeed a bug. Should be fixed now.